Dominik workshop - fixed same player selection bug, implemented sink transition and ready check for character select

DominikWaldowski5510 opened the pull request:
ready check currently has no art assets but works fine.
Required additional class to be added to properly support new UI element for controller.
Sink transition camera position can be adjusted by moving SinkCameraPoint transform found in main menu scene.

DominikWaldowski5510 committed:
d

d

DominikWaldowski5510 committed:
Merge branch 'Develop' into DominikWorkshop

DominikWaldowski5510 committed:
Added Ready check button, camera sink transition, fixed same character selection

Ready check has been added(currently lacking any art assets), added new class alongside it to register  UI properly with controller.

Camera sink transition has been added, to modify position of sink adjust position/rotation of sinkTransitionPoint found in main menu scene.

Fixed bug where same character could be picked if selected at same time by adding line of code in the check.

Danny-HypnoTank reviewed (changes requested):
Problem with loading to sink scene where it won't load, and speed and movement of transition needs to be changed

DominikWaldowski5510 committed:
merge fix attempt

merge fix attempt

Danny-HypnoTank committed:
Revert "merge fix attempt"

This reverts commit a412fd7889d9b34875ba2daa467f6e959f05f8fc, reversing
changes made to f1163ae4f0d439ab3310c9cd81cb077c4f8fb002.

DominikWaldowski5510 committed:
Medals Count and display(placeholders)

added counting for powerups/stuns/dashes and displayed them in end game screen.

DominikWaldowski5510 committed:
Merge branch 'DominikWorkshop' of https://github.com/Sturdge/Splash-Dash into DominikWorkshop

DominikWaldowski5510 committed:
Changed confirmation to ignore UI and only work with button press

Changed confirmation to ignore UI and only work with button press
